packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -241,3 +242,86 @@ class PointerEventConverter {
   static double _toLogicalPixels(double physicalPixels, double devicePixelRatio) =>
       physicalPixels == null ? null : physicalPixels / devicePixelRatio;
 }
+
+
+
+/// Serializes [ui.PointerData] to a json object.
+///
+/// Default values are omitted in the json object.
+///
+/// See [deserializePointerData].
+Map<String, dynamic> serializePointerData(ui.PointerData data) {
+      // to perform null check on a variable rather than a getter
```

Comment:
What is this comment referring to?

Reply:
This should have been removed. It was for null-safety when it was originally an attempt to patch engine.